qa/L0_sequence_corrid_batcher/sequence_corrid_batcher_test.py

```diff
@@ -38,6 +38,8 @@
 import numpy as np
 import sequence_util as su
 import test_util as tu
+import tritonclient.http as httpclient
+from tritonclient.utils import *
```

Comment:
What did you need imported from utils? Can you explicitly import what you need instead?